docs/integrations/platforms/certificate-agent.mdx

```diff
@@ -0,0 +1,421 @@
+---
+title: "Infisical Certificate Agent"
+description: "Learn how to use Infisical CLI Agent to manage certificates automatically."
+---
+
+The Infisical Certificate Agent is a specialized feature of the Infisical CLI designed to automatically manage certificates from your Infisical instance. It operates as a daemon process that continuously monitors and handles certificate lifecycle events including issuance, renewal, and storage without manual intervention.
+
+## Key features
+
+- **Automated Certificate Management**: Automatically issue, monitor, and renew certificates from Infisical [Certificate Profiles](/documentation/platform/pki/certificates/profiles)
+- **Certificate Signing Request (CSR) Support**: Submit pre-generated CSRs for certificate issuance
+- **Intelligent Renewal**: Configurable renewal thresholds and automatic certificate renewal before expiration
+- **Post-Event Hooks**: Execute custom commands after certificate issuance, renewal, or failure events
+- **Multiple Authentication Methods**: Support for Universal Auth, Kubernetes, AWS, GCP, and Azure authentication
+
+## Getting Started
+
+To use the Certificate Agent, you'll need:
+
+1. **Infisical CLI installed**: Download from [GitHub releases](https://github.com/Infisical/cli/releases)
+2. **Certificate Profile configured**: Set up a [certificate profile](/documentation/platform/pki/certificates/profiles) in your Infisical project
+3. **Authentication credentials**: [Machine identity](/documentation/platform/identities/overview) with appropriate permissions
+
+The agent operates using a YAML configuration file that defines authentication, certificate profiles, and operational parameters. When started, it authenticates with Infisical and begins monitoring certificates according to your configuration.
+
+## Certificate Management
+
+### Manual Certificate Issuance
+
+The agent will issue a certificate with specified parameters:
+
+```yaml
+version: v1
+
+certificates:
+  - profile-id: "prof-web-server-12345"
+    common-name: "api.company.com"
+    alt-names:
+      - "www.api.company.com"
+      - "api-v2.company.com"
+    key-algorithm: "RSA_2048"
+    signature-algorithm: "RSA-SHA256"
+    key-usages:
+      - "digital_signature"
+      - "key_encipherment"
+    extended-key-usages:
+      - "server_auth"
+    ttl: "30d"
+    file-output:
+      private-key-path: "./certs/api/private.key"
+      certificate-path: "./certs/api/certificate.crt"
+      file-permissions: "0600"
+```
+
+### CSR-based Issuance
+
+Submit a pre-generated [Certificate Signing Request](/documentation/platform/pki/certificates#guide-to-issuing-certificates):
+
+```yaml
+version: v1
+
+certificates:
+  - profile-id: "prof-external-ca-12345"
+    csr-path: "/etc/ssl/requests/api.csr"
+    file-output:
+      certificate-path: "/etc/ssl/certs/api.crt"
+      ca-certificate-path: "/etc/ssl/certs/ca.crt"
```

Comment:
Is this referring to the path at which the Root CA certificate should be stored?